transforms/no-implicit-this/helpers/plugin.js

```diff
@@ -89,7 +89,7 @@ function transform(root, options = {}) {
       return true;
     }
 
-    let helper = helpers.find(path => path.endsWith(name));
+    let helper = helpers.find(path => path.endsWith(`/${name}`));
```

Comment:
why the leading slash?

Reply:
because `'foo-bar-baz'.endsWith('baz') === true` 😉

Reply:
lol